stixcore/idb/tests/test_idb.py

```diff
@@ -17,10 +17,12 @@
 
 
 @pytest.fixture
+@pytest.mark.remote_data
```

Comment:
maybe in the test config we should download the IDB so this isn't necessary? seem awkward